practice-two/src/components/FormValidation/FormValidation.jsx

```diff
@@ -67,16 +116,28 @@ const FormValidation = (props) => {
       setErrors(newErrors);
 
       if (Object.keys(newErrors).length === 0) {
-        onShowForm();
+        const response = await createCustomerService(formData);
+
+        if (response.error) {
+          showToastInfo(MESSAGES.GET.ERRORS.ADD_FAILED);
```

Comment:
did you check all my message?
how about showToastInfo

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/1ea080d6-4c21-4c3e-a327-aa6b7fb98c3b)